Interview/demo.md

```diff
@@ -34,50 +34,62 @@ Before proceeding to the section of your choice, please consider the following t
 
 ### 1. The babylon demo project
 
-From a high level point of view the demo consists of a list of posts, where each post has its own detail.
+From a high level point of view the demo consists of a list of photos. Upon tapping any photo, the user is taken to the photo detail screen.
 
-#### Posts Screen
+#### Photos List Screen
 
-A post has a title and it's up to you how to display it. To retrieve the posts, you can use the following API:
+A Photo has a thumbnail image and a title and you can decide how best to display it. To retrieve the photos, you can use the following API:
 
-* http://jsonplaceholder.typicode.com/posts
+* http://jsonplaceholder.typicode.com/photos
 
-When a post is tapped, you should go to the detail screen.
+When a photo is tapped, you should go to the Photo detail screen.
+Also, a photo that is marked favourite shows up first in the list followed by other photos.
 
-#### Detail screen
+#### Photo Detail screen
 
-A post detail screen will have:
+A Photo detail screen will have:
 
-* Its author.
-* Its description (via the `body`).
-* The number of comments it has.
+* The user the photo belongs to.
+    * Get the user id from the album(http://jsonplaceholder.typicode.com/albums/{:id})
+    * Get the User from the user detail (http://jsonplaceholder.typicode.com/user/{:id})
+* The title of the photo.
+* The number of comments on the photo:
+    * (https://jsonplaceholder.typicode.com/photos/{:id}/comments)
+* The ability to mark/unmark a photo as favourite.
 
-You can retrieve the remaining information from these API:
 
-* http://jsonplaceholder.typicode.com/users
-* http://jsonplaceholder.typicode.com/comments
 
 #### Requirements
 
 The following requirements should be met:
 
 * Use Swift 4.0 or above.
-* The information (posts and post details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
-* Add reloading mechanism e.g. button or `UIRefreshControl`. `Reachability` is not required.
+* The information (photos and photo details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
+* A reloading mechanism e.g. button or `UIRefreshControl`. `Reachability` is not required.
 * **Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).**
 * The code should be production grade.
 * It should compile and run.
+* Stutter free, crash free UI.
+* If you are applying for Senior, there are additional Requirements:
+    * You should use a reactive framework
+    * UI should be written in code
+    * Self sizing Cells in TableView / CollectionView
```

Comment:
I'm afraid there is nothing to self-size anymore =) Title of the photo is very short